html_app/master_model.js

```diff
@@ -13956,16 +13972,16 @@ var __scb_sample_3 = {
                                 'cells': [
                                     {   'kind': 'checkbox',
                                         'name': 'gm6',
-                                        'treatment_id': 'growth_media'},
+                                        'treatment_id': 'growth_media6'},
```

Comment:
Just double-checking: is growth_media60 and growth_media6 supposed to be two different types of growth media?